lib/orders.rb

```diff
@@ -0,0 +1,41 @@
+require "order"
```

Comment:
I don't think that there are tests for this file.

Reply:
There is not, this was created as part of refactoring from the Cashier class so it should be covered by tests in the Cashier spec